snyk/client.py

```diff
@@ -131,8 +136,15 @@ def get(
         """
 
         path = cleanup_path(path)
-
-        url = f"{self.api_url}/{path}"
+        if version:
+            # When calling a "next page" link, it fails if a version parameter is appended on to the URL - this is a
+            # workaround to prevent that from happening...
+            if exclude_version:
```

Comment:
Pagination also supports version, if this is erroring on this endpoint lets chat with the team as it is unexpected. You should be able to grab the entire `next` link and send it as is with the same URL base

Reply:
Expected according to: https://snyk.slack.com/archives/C01JJUNL420/p1690465677963769